libs/common/src/enums/kdfType.ts

```diff
@@ -1,7 +1,12 @@
 export enum KdfType {
   PBKDF2_SHA256 = 0,
+  Argon2id = 1,
 }
 
+export const DEFAULT_ARGON2_MEMORY = 16;
+export const DEFAULT_ARGON2_PARALLELISM = 2;
+export const DEFAULT_ARGON2_ITERATIONS = 2;
```

Comment:
Should probably be updated to account for the latest changes here: https://github.com/OWASP/CheatSheetSeries/commit/f14bc4c0578e1328c23094dcb0a1d8e9b778b3ec

```suggestion
export const DEFAULT_ARGON2_MEMORY = 19;
export const DEFAULT_ARGON2_PARALLELISM = 2;
export const DEFAULT_ARGON2_ITERATIONS = 2;
```